src/pages/workspace/withPolicyConnections.tsx

```diff
@@ -0,0 +1,73 @@
+import {useEffect, useState} from 'react';
```

Comment:
Missing react import.

```suggestion
import React, {useEffect, useState} from 'react';
```

Reply:
added https://github.com/Expensify/App/pull/39132/commits/db3252e0aca375bd9f38fe40e2af66456135395a